plugins/org.teiid.designer.transformation.ui/src/org/teiid/designer/transformation/ui/wizards/rest/TeiidRestImportSourcePage.java

```
@@ -712,15 +713,27 @@ private File getXmlFileFromRestUrl(IConnectionProfile profile) {
 	private File convertJsonToXml(File jsonFile) throws IOException, Exception {
 
 		String jsonText = null;
+		boolean isArray = false;
+		String xml;
 		try {
 			jsonText = readFile(jsonFile);
 		} catch (IOException ex) {
```

It would be a good idea to fix the catch block here. If `jsonText = readFile(jsonFile);` ever throws an exception, it will result in an NPE exception down in the code. Might be a good idea to implement a proper error handling strategy here or just let it propagate to the caller.